Assets/Scripts/Auction/AuctionDriver.cs

```diff
@@ -0,0 +1,157 @@
+using CollectionExtensions;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+public class AuctionDriver : MonoBehaviour
+{
+    [SerializeField] private int numberOfItems;
```

Comment:
Maybe format declarations as
`
[SerializeField]

private int numberOfItems
`
instead, not that we have formally decided on using this semantic, so in the end it's up to you to decide 🤠

Reply:
resolved in [b68490b](https://github.com/hackerspace-ntnu/Prosjekt-Spill-H22/pull/20/commits/b68490b8af2b410452cf2b92baaa2d744269efce)